ddp-workspace/projects/ddp-singular/src/app/components/pages/participant-list/participant-list.component.ts

```diff
@@ -66,6 +70,7 @@ export class ParticipantsListComponent implements OnInit {
   ngOnInit(): void {
     this.clearParticipant();
     this.loadData();
+    console.log("#flag",this.featureFlag_DDP_8506);
```

Comment:
Do we need `console.log` here? (also see line 334)

Reply:
not really, good catch